Inc/CANProcess.h

```diff
@@ -15,9 +15,9 @@
 //defines for reading data from RxCanMsgTypeDef
 #define ID_PEDALBOX1							0x500
 #define ID_PEDALBOX2							0x501
-#define	ID_PEDALBOXCALIBRATE					0x503
```

Comment:
lets use astyle so we don't get stuff like this
http://astyle.sourceforge.net/

Reply:
i have implemented it in rinehart i think lol